src/frontends/tensorflow_common/src/pass/transpose_sinking.cpp

```diff
@@ -173,8 +173,8 @@ static void convert_binary_to_default_order(const shared_ptr<Node>& binary,
     // if right input is being implicitly broadcasted, insert a reshape
     // instead of a transpose
     shared_ptr<Node> new_node;
-    auto left_rank = get_static_rank(left);
-    if (left_rank < static_cast<int64_t>(perm_to_def.size()) && left.get_partial_shape().is_static()) {
+    auto left_rank = static_cast<size_t>(get_static_rank(left));
```

Comment:
should not we make `get_static_rank` return `size_t`?

Reply:
I will fix it.

Reply:
Done